scripts/go-dep.mk

```diff
@@ -0,0 +1,127 @@
+# Default configuration
+IMG_NAME := $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME)
+TAG := $(DOCKER_TAG)
+BASE_PKG := github.com/kyma-project/kyma
+IMG_GOPATH := /workspace/go
+IMG_GOCACHE := /root/.cache/go-build
+VERIFY_IGNORE := /vendor\|/automock
+
+# Other variables
+LOCAL_DIR = $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+COMPONENT_DIR = $(shell pwd)
+WORKSPACE_LOCAL_DIR = $(IMG_GOPATH)/src/$(BASE_PKG)/scripts
+WORKSPACE_COMPONENT_DIR = $(IMG_GOPATH)/src/$(BASE_PKG)/$(APP_PATH)
+FILES_TO_CHECK = find . -type f -name "*.go" | grep -v "$(VERIFY_IGNORE)"
```

Comment:
Can add a description to these variables. It would be fine to know the purpose of these without checking how they are used.